examples/dreamcast/parallax/bubbles/bubbles.c

```diff
@@ -133,6 +135,8 @@ static void sphere_frame_opaque(void) {
 
     plx_mat3d_pop();
 
+    plx_dr_finish();
```

Comment:
Here is needed? I'm not see `plx_dr_init` before.

Reply:
OH, SHIT. You're right!!! Good call!!